packages/terra-alert/docs/MoreInformation.md

```diff
@@ -0,0 +1,142 @@
+
+## Examples
+
+### Imports
+When importing the Alert component, it includes the Types object which contains a series of string constants that can be used for setting the 'type' prop for the component as shown in the examples below.
+```jsx
+import React from 'react';
+import Alert from 'terra-alert';
+```
+
+### Examples
+__________
+Basic Alert component of type 'alert' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.ALERT} >This is an alert</Alert>
+```
+or
+```jsx
+<Alert>This is an alert</Alert>
+```
+![Image](images/basicAlert.png?raw=true)
+__________
+Basic Alert component of type 'error' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.ERROR} >This is an error</Alert>
+```
+![Image](images/basicError.png?raw=true)
+__________
+Basic Alert component of type 'warning' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.WARNING} >This is a warning</Alert>
+```
+![Image](images/basicWarning.png?raw=true)
+__________
+Basic Alert component of type 'advisory' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.ADVISORY} >This is an advisory alert</Alert>
+```
+![Image](images/basicAdvisory.png?raw=true)
+__________
+Basic Alert component of type 'info' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.INFO} >This is an information alert</Alert>
+```
+![Image](images/basicInformation.png?raw=true)
+__________
+Basic Alert component of type 'success' using the default title and plain text for the message content.
+```jsx
+<Alert type={Alert.Opts.Types.SUCCESS} >This is a success alert</Alert>
+```
+![Image](images/basicSuccess.png?raw=true)
+__________
+Alert of type 'custom' with custom title, status color, and icon, and includes HTML in message content.
+```jsx
+<Alert type={Alert.Opts.Types.CUSTOM} 
+       title="Help!" 
+       customStatusColor="orange" 
+       customIcon={<IconHelp height="1.3333rem" width="1.3333rem" />} >
+  <span>This is a <u>custom</u> alert</span>
+</Alert>
+```
+![Image](images/customAlert.png?raw=true)
+__________
+Alert of type 'info' with custom title and long HTML content with show more/less toggle (using Terra Toggle).
+```jsx
+<Alert type={Alert.Opts.Types.INFO} title="Gettysburg Address:" >
+  <span>
+    Four score and seven years ago our fathers brought forth on this continent, 
+    a new nation, conceived in Liberty, and dedicated to the proposition that 
+    all men are created equal.
+    <Toggle isOpen={!this.state.allTextShown} isAnimated>
+      <Button onClick={this.handleShowMore} text="Show More" variant="link" size="tiny" />
```

Comment:
Could we change the "Show More" and "Show Less" toggle target from a Button.variant=link to actual link, or just change styling so would be `size="normal" (font-size: 1rem / 14px), text-decoration: underline, color: $terra-dark (#1c1f21), and no margins/padding left & right`

Also change Toggle to remove the `isAnimated` prop.